src/libDirectoryService/FinalBlockPreProcessing.cpp

```diff
@@ -164,13 +164,21 @@ void DirectoryService::ComposeFinalBlock() {
 
   StateHash stateRoot = AccountStore::GetInstance().GetStateRootHash();
 
+  // Compute the CommitteeHash member of the BlockHeaderBase
+  CommitteeHash committeeHash;
+  if (!Messenger::GetDSCommitteeHash(*m_mediator.m_DSCommittee,
+                                     committeeHash)) {
+    LOG_EPOCH(WARNING, to_string(m_mediator.m_currentEpochNum).c_str(),
+              "Messenger::GetDSCommitteeHash failed.");
```

Comment:
Return `false` here? (Like what is done for DS Block announcement)

Reply:
Okay. for now this func only returns void. Will add boolean return to interrupt the function at higher layer.

Reply:
Fixed.